sql/testdata/aggregate

```diff
@@ -238,6 +238,26 @@ SELECT k FROM kv HAVING k > 7;
 query error syntax error at or near ","
 SELECT COUNT(*, 1) FROM kv
 
+query I
+SELECT COUNT(*);
```

Comment:
[nit] we don't need the `;` in the test files. LGTM otherwise, thanks for the quick fix!